mongodb/collection.go

```diff
@@ -244,6 +244,23 @@ func (c *Collection) updateRecord(ctx context.Context, selector interface{}, upd
 	return nil, wrapMongoError(err)
 }
 
+// Replace replaces a single document located by the provided selector
+// The selector must be a document containing query operators and cannot be nil.
+// The newDocument cannot be nil or empty.
```

Comment:
should this be named replaceOne to follow the convention ?

Reply:
I'll rename it 👍